content/plugins/moritzebeling/kirby-headless-image-transformations/plugin.txt

```diff
@@ -0,0 +1,32 @@
+Title: Headless Image Transformations
+
+----
+
+Repository: https://github.com/moritzebeling/kirby-headless-image-transformations
+
+----
+
+Category: headless
+
+----
+
+Subcategory: images
```

Comment:
This subcategory currently doesn't exist. I'd suggest `api` as your plugin is essentially an API server for thumbnails.

Reply:
sure, makes sense